packages/snaps-controllers/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.2.0]
+### Added
+- Add support for links in Custom UI and Notifications ([#1814](https://github.com/MetaMask/snaps/pull/1814))
+
+### Fixed
+- Fix an issue with throwing snaps running long ([#1897](https://github.com/MetaMask/snaps/pull/1897))
```

Comment:
This sentence is confusing

Reply:
How about "Fix an issue where snaps throwing a SnapError would be allowed to run longer than expected"?